[feature] #286: link `api_spec` from the core repo (#287)
hyperledger/iroha-2-docs: src/api/index.md

<script setup>
import { useData } from 'vitepress'

const { page } = useData()
</script>

::: info

> This page contains the `api_spec.md` right from

Reviewer: ```suggestion
This page contains the copy of `api_spec.md` from
```

> This page contains the copy of `api_spec.md` from

Reviewer 2: ```suggestion
This page contains a copy of `api_spec.md` from
```

`hyperledger/iroha#iroha2-dev`. You can read the most up-to-date version of
it on
[GitHub](https://github.com/hyperledger/iroha/blob/iroha2-dev/docs/source/references/api_spec.md).

> Please note that this page was last updated at <b>{{ new Date(page.lastUpdated).toLocaleString() }}</b>.

Author: @6r1d, it supports, but AFAIK `markdown-it` will not parse `**{{ x }}**` as `<b>{{ x }}</b>`...

...or supports, you can check here: https://markdown-it.github.io/

Anyway, it doesn't matter much what syntax to use.

> Please note that this page was last updated at <b>{{ new Date(page.lastUpdated).toLocaleString() }}</b>.

Reviewer 2: ```suggestion
Please note this page was last updated on <b>{{ new Date(page.lastUpdated).toLocaleString() }}</b>.
```
